flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliClient.java

```diff
@@ -424,8 +424,14 @@ private void callReset(SqlCommandCall cmdCall) {
             else {
                 String key = cmdCall.operands[0].trim();
                 executor.resetSessionProperty(sessionId, key);
-                printRemovedAndDeprecatedKeyMessage(key);
-                if (!YamlConfigUtils.isRemovedKey(key)) {
+
+                boolean isRemovedKey = YamlConfigUtils.isRemovedKey(key);
+                boolean isDeprecatedKey = YamlConfigUtils.isDeprecatedKey(key);
+                if (isRemovedKey || isDeprecatedKey) {
+                    printRemovedAndDeprecatedKeyMessage(key);
+                }
+                // it's not removedKey, need to print info message
+                if (!isRemovedKey) {
```

Comment:
nit: Why don't use 
```
if (...) {

} else {
...
}

```

It seems deprecated key will also get into this if-block.

Reply:
because when isDeprecatedKey is true , it need to print  a  deprecated message  in addition to the normal message.